taskcluster/script.py

```diff
@@ -32,6 +31,52 @@ def fatal(message):
     print('TEST-UNEXPECTED-FAIL | bitbar | {}'.format(message))
     sys.exit(TBPL_RETRY_EXIT_STATUS)
 
+def enable_charging(device):
+    rc = 0
+    timeout = 10
+    p2_path = "/sys/class/power_supply/battery/input_suspend"
+    g5_path = "/sys/class/power_supply/battery/charging_enabled"
+
+    try:
+        device_name = device.shell_output("getprop ro.product.model", timeout=timeout)
+        if device_name == "Pixel 2":
+            p2_batt_input_suspended = (
+                device.shell_output(
+                    "cat %s 2>/dev/null" % p2_path, timeout=timeout
+                ).strip()
+                == "1"
+            )
+            if p2_batt_input_suspended:
+                print("Enabling charging...")
+                device.shell_bool(
+                    "echo %s > %s" % (0, p2_path), root=True, timeout=timeout
+                )
+        elif device_name == "Moto G (5)":
+            g5_charging_enabled = (
+                device.shell_output(
+                    "cat %s 2>/dev/null" % g5_path, timeout=timeout
+                ).strip()
+                == "1"
```

Comment:
Shouldn't this be "0" ?

Reply:
I think it's right. The path for the g5 is called "charging_enabled". I check if it's one and set the boolean g5_charging_enabled true if it is. On line 61, I check that the boolean is _not_ set (the device is not charging), and then set it to 1.